.github/workflows/auto-review-comment.yml

```diff
@@ -0,0 +1,34 @@
+name: Auto Review Comment
+
+on:
+  workflow_run:
+    workflows:
+      - Diff Compile
```

Comment:
Where do we have this worflow?

Reply:
I am planning to add it in the next PR, it doesn't matter for now imo

Reply:
Got it. Let’s rename if to `Auto-review Prepare` to match other prepare-do workflows.

Reply:
Or `Auto-review Diff Prepare`